pkg/registry/apis/provisioning/jobs/migrate/legacy_resources.go

```diff
@@ -108,6 +108,7 @@ type legacyResourceResourceMigrator struct {
 	options   provisioning.MigrateJobOptions
```

Comment:
good rename. I restrained myself to do that in the previous PR because it was already a lot.